<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="cbJoystickInterface">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Cursor Interface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Kempston Interface</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinclair Interface 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sinclair Interface 2 (port 2)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="QTabWidget" name="twControls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Keyboard</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="KeyboardWidget" name="keyboard" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Joystick</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="4">
         <widget class="QPushButton" name="pbFire">
          <property name="text">
           <string>Огонь</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbUp">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Вверх</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pbRight">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Впавро</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pbDown">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Вниз</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>201</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbLeft">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Влево</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="cbCaptureKeyboard">
      <property name="text">
       <string>Capture keyboard</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cbShowControls">
      <property name="text">
       <string>Show controls</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnReset">
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="ScreenWidget" name="screen" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuScreenshot">
     <property name="title">
      <string>Screenshot</string>
     </property>
     <addaction name="actionLoad"/>
     <addaction name="actionSave"/>
    </widget>
    <addaction name="action_Load_a_snapshot"/>
    <addaction name="actionLoad_a_Z80"/>
    <addaction name="actionSave_a_Z80_snapshots"/>
    <addaction name="separator"/>
    <addaction name="menuScreenshot"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Machine">
    <property name="title">
     <string>&amp;Machine</string>
    </property>
    <widget class="QMenu" name="menuPalette">
     <property name="title">
      <string>Palette</string>
     </property>
     <addaction name="actionPalette_1"/>
     <addaction name="actionPalette_2"/>
     <addaction name="actionPalette_3"/>
    </widget>
    <addaction name="action_Reset"/>
    <addaction name="actionNMI"/>
    <addaction name="separator"/>
    <addaction name="menuPalette"/>
    <addaction name="separator"/>
    <addaction name="actionSpectrum_128k"/>
    <addaction name="actionSpectrum_48k"/>
    <addaction name="actionOpenSE_Basic_48k"/>
    <addaction name="actionArcade_Game_Designer_48k"/>
    <addaction name="actionSinclair_Test_ROM_48k"/>
    <addaction name="separator"/>
    <addaction name="actionOpenRom"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Machine"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Load_a_snapshot">
   <property name="text">
    <string>&amp;Load a SNA snapshot</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="action_NMI">
   <property name="text">
    <string>&amp;NMI</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionSpectrum_128k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Spectrum 128k</string>
   </property>
  </action>
  <action name="actionSpectrum_48k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Spectrum 48k</string>
   </property>
  </action>
  <action name="actionPalette_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Palette 1</string>
   </property>
  </action>
  <action name="actionPalette_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Palette 2</string>
   </property>
  </action>
  <action name="actionPalette_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Palette 3</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionNMI">
   <property name="text">
    <string>NMI</string>
   </property>
  </action>
  <action name="actionLoad_a_Z80">
   <property name="text">
    <string>Load a Z80 snapshot</string>
   </property>
  </action>
  <action name="actionSave_a_Z80_snapshots">
   <property name="text">
    <string>Save a Z80 snapshots</string>
   </property>
  </action>
  <action name="actionOpenRom">
   <property name="text">
    <string>Open .ROM file</string>
   </property>
  </action>
  <action name="actionArcade_Game_Designer_48k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Arcade Game Designer</string>
   </property>
  </action>
  <action name="actionOpenSE_Basic_48k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OpenSE Basic</string>
   </property>
  </action>
  <action name="actionSinclair_Test_ROM_48k">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sinclair Test ROM</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScreenWidget</class>
   <extends>QWidget</extends>
   <header location="global">screenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KeyboardWidget</class>
   <extends>QWidget</extends>
   <header location="global">keyboardwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
